resolwe_bio/tools/make_igv_session_archive.py

```diff
@@ -8,28 +8,49 @@
 
 parser = argparse.ArgumentParser(description="Create igv session.")
 
-parser.add_argument('-f', '--input_file', required=True, help="File with paths to files for IGV")
+parser.add_argument('-f', '--input_file', required=True, help="File with paths to files for IGV.")
 
 args = parser.parse_args()
 
 
-def make_xml_tree(input_file):
+def get_build_info(input_file):
+    """Get build information in UCSC notation, if possible."""
+    build_dict = {
+        'hg38': ['GRCh38'],
+        'hg19': ['GRCh37', 'b37'],
+        'mm10': ['GRCm38'],
+        'mm9': ['MGSCv37'],
+        'rn6': ['Rnor_6.0'], }
+
+    _, build, _, _ = input_file.split('_')
+    new_build = [k for k, v in build_dict.items() if k == build or build.startswith(tuple(v))]
+    if new_build:
+        return new_build[0]
+    else:
```

Comment:
You can replace::

    else:
        return ''

With just:

    return ''